tests/baselines/reference/jsdocTemplateConstructorFunction2.errors.txt

```diff
@@ -4,8 +4,8 @@ tests/cases/conformance/jsdoc/templateTagWithNestedTypeLiteral.js(26,15): error
 
 ==== tests/cases/conformance/jsdoc/templateTagWithNestedTypeLiteral.js (2 errors) ====
     /**
-     * @template {T}
      * @param {T} t
+     * @template {T}

```

Comment:
Did this need to get reordered in the input file?

Reply:
I reordered it in the input file to show that `@template` works before or after. I was talking about it with @mhegazy and found that we had completely opposite intuitions about whether the usual order is before or after.